Build/Build.cs

```diff
@@ -128,13 +171,33 @@ class Build : NukeBuild
         .DependsOn(ApiChecks)
         .DependsOn(TestFrameworks)
         .DependsOn(UnitTests)
+        .DependsOn(CalculateNugetVersion)
         .Executes(() =>
         {
             DotNetPack(s => s
                 .SetProject(Solution.Core.FluentAssertions)
                 .SetOutputDirectory(ArtifactsDirectory)
                 .SetConfiguration("Release")
                 .EnableContinuousIntegrationBuild() // Necessary for deterministic builds
-                .SetVersion(GitVersion.NuGetVersionV2));
+                .SetVersion(SemVer));
+        });
+
+    Target Push => _ => _
+        .DependsOn(Pack)
+        .OnlyWhenDynamic(() => IsTag)
+        .Executes(() =>
+        {
+            var packages = GlobFiles(ArtifactsDirectory, "*.nupkg");
+
+            DotNetNuGetPush(s => s
+                .SetApiKey(ApiKey)
+                .EnableSkipDuplicate()
+                .SetSource("https://api.nuget.org")
```

Comment:
Have you tried that this url works as source?

These two places use the full url `"https://api.nuget.org/v3/index.json"`.
https://github.com/nuke-build/nuke/blob/bcd5968c41b2a4c21eacfa6b336230971988adb9/source/Nuke.GlobalTool/templates/Build.cs#L39

https://github.com/nuke-build/nuke/blob/bcd5968c41b2a4c21eacfa6b336230971988adb9/source/Nuke.Components/IPublish.cs#L20

Reply:
Oh crap. I changed it to experiment with the build.